examples/rl/policy_gradient/rl_policy_gradient_solver.py

```diff
@@ -0,0 +1,146 @@
+import time
+import pickle
+import os
+
+import minpy.numpy as np
+from minpy import core
+from minpy.nn.solver import Solver
+
+class RLPolicyGradientSolver(Solver):
+    """A custom `Solver` for policy gradient models.
+    Specifically, the model should provide:
+        .forward(X)
+        .choose_action(p)
+        .loss(xs, ys, rs)
+        .discount_rewards(rs)
+        .preprocessor
+    """
+    def __init__(self, model, env, **kwargs):
```

Comment:
Could you please add more comments on what's in the kwargs and what's the meaning of them? Thanks.